### PR TITLE
Write more about that rain forest worth of paperwork

### PR DESCRIPTION
**Pull request by jgorset**
I'm getting pretty good at this.

### CODE REVIEW

File: YOU_GOT_HIRED.md [hunk 1]
```diff
@@ -20,10 +20,15 @@ of experience with hiring awesome people.
...
-The Airport Express ("Flytoget" in Norwegian) or the regular train. The Airport Express is slightly faster
-and slightly more expensive. Either way you should get off at Oslo Sentralbanestasjon. Just let us know
-when you board the train and we'll meet you there!
+Whoa there cowboy! If you're from a country outside the EU/EEA, you'll have to apply for a visitor's
+visa. This process varies depending on where you're from, but there's an excellent guide on the
+[website of The Norwegian Directorate of Immigration (UDI)](http://www.udi.no/en/want-to-apply/visit-and-holiday/)
+that will walk you through the specifics.
+
+Once your visa is in order (or if you don't need one), you'll want to get a flight to Oslo Airport Gardermoen
+(OSL) and catch a train into town. There's two; The Airport Express ("Flytoget" in Norwegian) or the regular
```
Comment (NSElvis) [inline YOU_GOT_HIRED.md]: Should this be `There's two:` ? (:)

Comment (jgorset) [inline YOU_GOT_HIRED.md]: Yes, thank you — nice catch!

**Comment by NSElvis** [comment]
Might be better to include a few links 

```markdown
[The Airport Express](http://www.flytoget.no/eng/)

[Oslo Sentralstasjon](https://www.google.no/maps/place/Oslo+Sentralstasjon/@59.911032,10.752408,17z)
```

**Comment by jgorset** [comment]
Good idea!

**Comment by NSElvis** [comment]
>(otherwise we recommend watching 10 hours of badgers the requisite number of times).

I don't understand `the requisite number of times` :( mind explaining?

**Comment by jgorset** [comment]
However many times it takes to watch it for your application to be processed. Maybe I should rephrase — but to what?

**Comment by NSElvis** [comment]
>(otherwise we recommend you watching badgers non-stop until your application gets processed)

**Comment by timkurvers** [comment]
Either:

> (otherwise we recommend watching badgers non-stop until your application gets processed)

Or:

> (otherwise we recommend you to watch badgers non-stop until your application gets processed)

**Comment by sindrenm** [comment]
@jgorset: You're getting _too_ good at this.

**Pull request merged**